### DISCUSSION TITLE
My Github Activity Feed isn't displaying correctly

### DISCUSSION THREAD

**Discussion by peoray** [community/community#22376] [New to GitHub]
<p>My Github Feed Activity isn’t displaying correctly. When I go to <a href="http://github.com" rel="noopener nofollow ugc">github.com</a>. The activity feed does not load accurate data. For the past two weeks or so, the data has been irregular. For example, some data has been persistent for the past two weeks and still showing 5hours ago. I can see only eight items on my feed and the ‘load more’ button is not visible. Please what’s going on??</p>
<p>Go to <a href="http://github.com" rel="noopener nofollow ugc">github.com</a> - on the activity feed, you’re supposed to see a chronological view of all activities by the people you’re following.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/d/6/d602d458fffd56e57d748e7fd9387fa0107d50c6.jpeg" data-download-href="/uploads/short-url/uxeeVH5fniKdbgbVMw1ofAsA2EK.jpeg?dl=1" title="Screen Shot 2021-12-06 at 1.06.32 PM" rel="noopener nofollow ugc"><img src="https://user-images.githubusercontent.com/23735423/181070647-064e0ed1-89b9-4500-9ae5-71219f200abb.jpeg" alt="Screen Shot 2021-12-06 at 1.06.32 PM" data-base62-sha1="uxeeVH5fniKdbgbVMw1ofAsA2EK" width="593" height="499" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d602d458fffd56e57d748e7fd9387fa0107d50c6_2_593x499.jpeg, https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d602d458fffd56e57d748e7fd9387fa0107d50c6_2_889x748.jpeg 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d602d458fffd56e57d748e7fd9387fa0107d50c6_2_1186x998.jpeg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/d/6/d602d458fffd56e57d748e7fd9387fa0107d50c6_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">Screen Shot 2021-12-06 at 1.06.32 PM</span><span class="informations">1910×1610 119 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by jfullstackdev** [comment, marked as answer]
<p>it’s because of the one the you follow, he followed 228+ in just a single moment, I’m experiencing that also from time to time, your newsfeed will not be displaying that much because of that,</p>
